recipes/getting-started/getting_started-image_semantic_segmentation.ipynb

```diff
@@ -0,0 +1,824 @@
+    "\n",
+    "In image semantic segmentation our goal is not only to know of the presence of given objects, but also to identify their positions, dimensions and shape. This require work at pixel level in order to classify every single pixel of the object. This can be achieved in an approximate way by drawing the shape of the object.\n",
+    "\n",
+    "In the image below you'll find an example of Kili's interface for image semantic segmentation for an image of a Tesla car.\n",
+    "\n",
+    "![](../img/getting_started/semantic-segmentation.png)\n",
+    "\n",
+    "Using image object detection can be very helpful in domains such as face recognition, medicine, self-driving cars, etc. It's specially interesting to be able to automate this process, however to achieve good performance it's generally necessary to have a lot of labeled data. A great way to label your data efficiently is using Kili. The next steps of this tutorial will walk you through the creation of your first image object detection project at Kili."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connecting to Kili <a id='connect'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first step is to be able to connect to the platform.\n",
+    "\n",
+    "If you use the SaaS version of Kili (see [here](https://cloud.kili-technology.com/docs/hosting/saas/)), you use by default the Auth0 login identification, or your company's authentication if it has been implemented. \n",
+    "\n",
+    "<img src=\"../img/auth0.png\" width=\"400\" />\n",
+    "\n",
+    "If you use Kili on premise (see [here](https://cloud.kili-technology.com/docs/hosting/on-premise-entreprise/)), you will probably use our own authentication :\n",
+    "\n",
+    "<img src=\"../img/noauth0.png\" width=400 />\n",
+    "\n",
+    "You need your organization admin to create your profile, and depending on the authentication implementation, you can sign up and set your password, or use the temporary one provided to you by the admin. \n",
+    "\n",
+    "If everything succeeds you should arrive at the projects page shown in the beginning of the next section."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating the project <a id='project'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### List of projects"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You arrive on a list of projects.\n",
+    "\n",
+    "![](../img/project_list.png)\n",
+    "\n",
+    "You can refer to this [document](https://cloud.kili-technology.com/docs/definitions/definitions/) to find the definitions of key concepts at Kili. One of them is a project : it is a combination of :\n",
```

Comment:
In English, no space in punctuation before ` :`, ` !`, etc (check everywhere in the PR))